sylt-common/src/ty.rs

```diff
@@ -84,6 +86,10 @@ impl Hash for Type {
             Type::ExternFunction(_) => {
                 16
             }
+
+            Type::Invalid => {
+                19
+            }
```

Comment:
I don't like that these numbers are un-ordered.

Reply:
I have a todo to make the enum "countable" - that would remove the entire match block and would make me happy. I might do that now actually.

Reply:
See #255

Reply:
See #255